runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -0,0 +1,38 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
+/**
+ * Created by maithem on 7/28/17.
+ */
+@Data
+@AllArgsConstructor
+public class MultipleReadRequest implements ICorfuPayload<MultipleReadRequest> {
```

Comment:
A short class level comment (nothing fancy for this) would be good. Also we should discuss and have a high level header in all the class files. This is not for this PR though.